ci/ci/constants.py

```diff
@@ -29,4 +29,5 @@ def __init__(self, gh_username: str, hail_username: Optional[str] = None, teams:
     User('jkgoodrich', 'jgoodric'),
     User('konradjk', 'konradk'),
     User('patrick-schultz', 'pschultz', HAIL_TEAM),
+    User('grohli', 'grohlice'),
```

Comment:
I suggest adding yourself to `SERVICES_TEAM` at least

Reply:
Done, and I've placed myself in alphabetical order too.